modules/olm-rukpak-bundle.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm-packaging-format.adoc
+
+:_content-type: CONCEPT
+[id="olm-rukpak-bundle_{context}"]
+= Bundle
+
+A RukPak `Bundle` object represents content that can be made available to other consumers in the cluster. Much like the contents of a container image must be pulled and unpacked in order for pod to start using them, `Bundle` objects are used to reference content that might need to be pulled and should be unpacked. In this sense, a bundle is a generalization of the image concept and can be used to represent any type of content.
```

Comment:
Suggestions as long as they don't change the meaning

```suggestion
A RukPak `Bundle` object represents content to make available to other consumers in the cluster. Much like the contents of a container image must be pulled and unpacked in order for pod to start using them, `Bundle` objects are used to reference content that might need to be pulled and unpacked. In this sense, a bundle is a generalization of the image concept and can be used to represent any type of content.
```

Reply:
There was a slight technical difference meaning between the "pulled" and then "unpacked", but I think it's fine to smooth out / not worth making the distinction.